eth/tracers/api_blocktrace.go

```diff
@@ -306,7 +326,10 @@ func (api *API) getTxResult(env *traceEnv, state *state.StateDB, index int, bloc
 		addrStr := addr.String()
 
 		env.pMu.Lock()
-		_, existed := env.Proofs[addrStr]
+		checkedProof, existed := env.Proofs[addrStr]
+		if existed {
+			txStorageTrace.Proofs[addrStr] = checkedProof
```

Comment:
So we might have lots of duplicate account proofs for different transactions, right?

Reply:
yes and it's on purpose